dowhy/timeseries/causal_effect_tutorial.ipynb

```diff
@@ -77,12 +73,12 @@
    ],
    "source": [
     "# Prompt user to input the path to the CSV file\n",
-    "file_path = \"./temporal_graph.csv\"\n",
+    "file_path = \"../../docs/source/example_notebooks/datasets/temporal_graph.csv\"\n",
     "\n",
     "# Create the graph from the CSV file\n",
     "graph = create_graph_from_csv(file_path)\n",
     "pretty_print_graph(graph)\n",
-    "display_networkx_graph(graph)"
+    "plot(graph)"
```

Comment:
for illustration, better to choose a connected graph? right now, it feels as if two graphs are being output